notebooks/250-fastcomposer-image-generation/250-fastcomposer-image-generation.ipynb

```diff
@@ -0,0 +1,1352 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !git clone https://github.com/mit-han-lab/fastcomposer.git
</pre><p>what if notebook runs not n first time and repo already cloned?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>